tests/Support/AuthorRule.php

```diff
@@ -12,8 +12,8 @@
  */
 final class AuthorRule implements RuleInterface
 {
-    public function execute(string $userId, Item $item, array $parameters = []): bool
+    public function execute(?string $userId, Item $item, array $parameters = []): bool
     {
-        return $parameters['authorID'] === $userId;
+        return isset($parameters['authorID']) && $parameters['authorID'] === $userId;
```

Comment:
`isset(null) === false`, so when `$userId` is null and `$parameters['authorID']` is null this code returns false. Is it expected?

Reply:
Yes.